.github/workflows/conformance.yml

```diff
@@ -42,10 +42,8 @@ jobs:
       - uses: sigstore/sigstore-conformance@ee4de0e602873beed74cf9e49d5332529fe69bf6 # v0.0.11
         with:
           entrypoint: ${{ github.workspace }}/conformance
-          skip-signing: true
 
       - uses: sigstore/sigstore-conformance@ee4de0e602873beed74cf9e49d5332529fe69bf6 # v0.0.11
         with:
           entrypoint: ${{ github.workspace }}/conformance
```

Comment:
Just wondering, I assume that some tests will still fail because we don't have conformance hooked up for DSSE signing, correct? How does this surface?

Reply:
This is my first time really digging in the signing side of conformance tests. We do [test DSSE verification](https://github.com/sigstore/sigstore-conformance/blob/main/test/test_bundle.py#L33-L44) but I don't think we test DSSE signing today, since there [isn't a way to specify it in the CLI protocol](https://github.com/sigstore/sigstore-conformance/blob/main/docs/cli_protocol.md).

I created https://github.com/sigstore/sigstore-conformance/issues/145 to track this request.

Reply:
Ah! This actually wasn't what I was asking originally, but I'm glad this was noticed. I was thinking that because in our conformance test, we are only setting up message signing, we would see a failure and need an XFAIL.